config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root to: 'application#index'
+
+  resources :drivers
+  resources :passengers do
+    resources :trips, only: [:index, :new, :create]
```

Comment:
I'm not convinced you need `:index` or `:new` on these -- let me know if you have questions!